### PR TITLE
more jigsaw tests

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

Adds a polradec... etc. tests. 

## Motivation and Context

bringing down 9GB of jigsaw tests. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/tests/FunctionalTestsHicrop.cpp [hunk 1]
```diff
@@ -388,6 +388,7 @@ TEST_F(MroCube, FunctionalTestHicropInstrumentError) {
 
   try {
      hicrop(testCube, options, logs);
+     FAIL() << "Exepected Error thrown";
```
Comment (acpaquette) [inline isis/tests/FunctionalTestsHicrop.cpp:391]: These could maybe go in another PR but are fine either way.

**Review by acpaquette — COMMENTED** [review]
Overall looks good. :+1: I would like either @jessemapel or @kberryUSGS to take a look over the tests to confirm that they are basically testing what they were before.

File: isis/tests/Fixtures.cpp [hunk 2]
```diff
@@ -333,89 +302,19 @@ namespace Isis {
...
-    if (cube4) {
-      delete cube4; 
-    }
-
-    if (cube5) {
-      delete cube5; 
-    }
+    for(int i = 0; i < cubes.size(); i++) {
+      if(cubes[i] && cubes[i]->isOpen()) {
+        cubes[i]->close();
```
Comment (jessemapel) [inline isis/tests/Fixtures.cpp]: You should just delete cubes[i] if it's non-null. The destructor will close the cube for you.

File: isis/tests/Fixtures.cpp [hunk 3]
```diff
@@ -277,53 +277,22 @@ namespace Isis {
...
-    cubeList->append(cube5->fileName());
-    cubeList->append(cube6->fileName());
-    cubeList->append(cube7->fileName());
-
+    
+    for(int i = 0; i < isdFiles.size(); i++) {
+      int n = i+1; // filenames use 1 based indexing 
+      isdFiles[i] = new FileName("data/apolloNetwork/apolloImage"+QString::number(n)+".isd");     
+      labelFiles[i] = new FileName("data/apolloNetwork/apolloImage"+QString::number(n)+".pvl");
+      cubes[i] = new Cube();
```
Comment (jessemapel) [inline isis/tests/Fixtures.cpp:289]: Why not avoid the memory management issues and just start the vectors out empty and then fill them with the actual objects instead of just pointers?

File: isis/tests/FunctionalTestsJigsaw.cpp [hunk 4]
```diff
@@ -808,3 +852,254 @@ TEST_F(ObservationPair, FunctionalTestJigsawErrorTBParamsNoSolve) {
...
+
+  EXPECT_THAT(lines[75].toStdString(), HasSubstr("RADII: TRIXIAL"));
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, lines[76].trimmed(), "");
+  
+  // check residual stats 
+  EXPECT_THAT(lines[132].toStdString(), HasSubstr("minimum: -178.872"));
+  EXPECT_THAT(lines[133].toStdString(), HasSubstr("Quartile 1:   -9.638"));
+  EXPECT_THAT(lines[134].toStdString(), HasSubstr("Median:   +1.377"));
+  EXPECT_THAT(lines[135].toStdString(), HasSubstr("Quartile 3:   +9.449"));
+  EXPECT_THAT(lines[136].toStdString(), HasSubstr("maximum: +175.731"));
```
Comment (jessemapel) [inline isis/tests/FunctionalTestsJigsaw.cpp]: I would expect these two tests to check the values put in the bundlout.txt file by the BundleTargetBodty object, see https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/control/objs/BundleUtilities/BundleTargetBody.cpp#L958 these are the actual values we are solving for.

**Review by jessemapel — APPROVED** [review]
